process/sync/baseSync.go

```diff
@@ -1238,6 +1243,29 @@ func (boot *baseBootstrap) handleTokensSuppliesRepopulation() error {
 	return tokensSuppliesProc.SaveSupplies()
 }
 
+func (boot *baseBootstrap) syncAccountsDBs(key []byte, id string) error {
+	// TODO: refactor this in order to avoid treatment based on identifier
+	switch id {
+	case dataRetriever.UserAccountsUnit.String():
+		log.Error("UserAccountsUnit")
+		return boot.syncUserAccountsState(key)
+	case dataRetriever.PeerAccountsUnit.String():
+		log.Error("PeerAccountsUnit")
+		return boot.syncValidatorAccountsState(key)
+	default:
+		return fmt.Errorf("invalid trie identifier, id: %s", id)
+	}
+}
+
+func (boot *baseBootstrap) syncShardAccountsDBs(key []byte, _ string) error {
+	return boot.syncUserAccountsState(key)
+}
+
+func (boot *baseBootstrap) syncValidatorAccountsState(key []byte) error {
+	log.Warn("base sync: started syncValidatorAccountsState")
```

Comment:
no log warn.

Reply:
This was actually copy-pasted from original code when syncing data on meta, wanted to leave it as it is.